### PR TITLE
fix: image wrapper width in notifications

### PR DESCRIPTION
**Pull request by omBratteng**
### Describe what this PR does

Fixes the width of video image in notifications, and reduces the size of the play icon

<table>
<tr>
 <td>Before
 <td>After
<tr>
 <td>
image
</td>
 <td>image
</table>

### CODE REVIEW

File: packages/shared/src/components/image/VideoImage.tsx [hunk 1]
```diff
@@ -7,18 +7,21 @@ import { ImageProps, Image } from './Image';
 export interface VideoImageProps extends ImageProps {
   size?: IconSize;
   wrapperClassName?: string;
+  wrapperWidth?: 'w-full' | 'w-fit';
```
Comment (rebelchris) [inline packages/shared/src/components/image/VideoImage.tsx:10]: Isn't that what the wrappedClassName should do?

**Review by rebelchris — CHANGES_REQUESTED** [review]
I'm not 100% sure why we needed the change, maybe check with the person who originally added it what it supposed to solve.
Else we go back and forth between issues maybe.

File: packages/shared/src/components/image/VideoImage.tsx (hunk 1 above)
Comment (omBratteng) [reply packages/shared/src/components/image/VideoImage.tsx:10]: First thought about adding `!w-fit` to wrapperClassName, but then you get an extra classname.

But I will do some more testing and see if w-full is actually needed in production

Comment (rebelchris) [reply packages/shared/src/components/image/VideoImage.tsx:10]: Cool yeah please check as I'm not sure for which place it was added as it's a generic component basically.

Comment (omBratteng) [reply packages/shared/src/components/image/VideoImage.tsx:10]: The change from w-fit to w-full was in a different component, done in #2436.

If I set it to be w-fit always, the normal cards looks wrong
image

Comment (omBratteng) [reply packages/shared/src/components/image/VideoImage.tsx:10]: Though, I think I actually can just add an empty div around the imagecomponent here to fix it.

File: packages/shared/src/components/image/VideoImage.tsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ export interface VideoImageProps extends ImageProps {
 }
 
 const VideoImage = ({
-  size = IconSize.XXXLarge,
+  size = IconSize.XLarge,
```
Comment (nensidosari) [inline packages/shared/src/components/image/VideoImage.tsx:13]: Does this mix up the size for the video images on cards? I think they require the XXXLarge by default?

**Review by nensidosari — APPROVED** [review]
LGTM as a solution for the moment, though not a big fan of unnecessary wrapping divs only for layout fixes. 
Maybe some solution can be given modifying the already wrapping div of notification, but we can leave that for future improvement if we want this fix shipped fast.

**Review by rebelchris — APPROVED** [review]
Let's give it a go, but monitor impact :)

**Pull request merged**